### PR TITLE
fixed symbol text position to adjust for large schematic symbols

### PR DESCRIPTION
**Pull request by ShiboSoftwareDev**
for the missing symbols, I need to update core first, however it needs the schematic-symbols repo updated and also this repo for testing. I tested with yalc and it worked, I'll get back and fix the tests after core is merged.

/claim #154
image

### CODE REVIEW

File: lib/sch/svg-object-fns/create-svg-objects-from-sch-component-with-symbol.ts [hunk 1]
```diff
@@ -173,21 +173,57 @@ export const createSvgObjectsFromSchematicComponentWithSymbol = ({
...
+      const transformScale = Math.abs(transform.a)
+
+      if (anchor.includes("bottom")) {
+        return baseOffset * transformScale
+      }
+      if (anchor.includes("top")) {
+        return -baseOffset * transformScale
+      }
+      return 0
+    }
```
Comment (seveibar) [inline lib/sch/svg-object-fns/create-svg-objects-from-sch-component-with-symbol.ts:203]: Does this function need to be nested?

**Review by seveibar — CHANGES_REQUESTED** [review]
Can you use pkg.pr.new with your core branch?

**Comment by ShiboSoftwareDev** [comment]
> Can you use pkg.pr.new with your core branch?

I don't know what that means

File: lib/sch/svg-object-fns/create-svg-objects-from-sch-component-with-symbol.ts (hunk 1 above)
Comment (ShiboSoftwareDev) [reply lib/sch/svg-object-fns/create-svg-objects-from-sch-component-with-symbol.ts:203]: it doesn't I'll flatten it

**Comment by seveibar** [comment]
Make a pr to core with your required core changes, then look at the actions. One of the actions should give you an installation command

File: tests/sch/transform-attribute.test.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { test, expect } from "bun:test"
 import { convertCircuitJsonToSchematicSvg } from "lib"
 
-test("svg should have data-real-to-screen-transform attribute", () => {
+test.skip("svg should have data-real-to-screen-transform attribute", () => {
```
Comment (seveibar) [inline tests/sch/transform-attribute.test.tsx:4]: Dont skip this, just run bun upgrade

Comment (ShiboSoftwareDev) [reply tests/sch/transform-attribute.test.tsx:4]: I forgot to remove the skip

**Comment by ShiboSoftwareDev** [comment]
@seveibar I'm using the stackblitz version of core, what now?

**Comment by seveibar** [comment]
@ShiboSoftwareDev shouldnt using your branch via stackblitz fix the snapshots? They still look broken

**Comment by seveibar** [comment]
We're going to stop allowing broken snapshots is the tldr

**Comment by seveibar** [comment]
Another way to avoid the dance is to make it so schematic symbols is backwards compatible, whoever fid the resistor thing made a change that wasnt backwards compatible

**Comment by ShiboSoftwareDev** [comment]
they got fixed except one, let me double check

**Comment by ShiboSoftwareDev** [comment]
@seveibar the rotated resistor test was strange, other than that all looks good

**Comment by ShiboSoftwareDev** [comment]
@seveibar what happens now?

**Review by seveibar — APPROVED** [review]

**Pull request merged**